ansible/group_vars/all

```diff
@@ -303,16 +303,16 @@ openwhisk_cli_tag: "{{ lookup('ini', 'git_tag section=openwhisk-cli file={{ open
 # remote mode or location to save the binaries of the OpenWhisk CLI in local mode.
 #
 
+installation_mode: remote
```

Comment:
This should not be outside top level, it should be inside openwhisk_cli object.

Reply:
Now that I look what you are doing I think we need more specific name and also a way to override via en environment.
How about this:
```
cli_installation_mode_default: local
cli_installation_mode: "{{ openwhisk_cli_installation_mode | default(cli_installation_mode_default) }}"
```
Then for example ins `environments/local/group_vars/all it can override via
```
openwhisk_cli_installation_mode: remote
```
In addition via cmd flag
```
ansible-playbook -i environments/local openwhisk.yml -e cli_installation_mode=remote;  to download binaries.

ansible-playbook -i environments/local openwhisk.yml -e cli_installation_mode=local; to copy local binaries.
```

Reply:
Set the default to `local` we are not ready for `remote`, the `remote` will be together with the remove of cli source